src/map.c

```diff
@@ -1439,21 +1436,19 @@ void map_renderCommod( double bx, double by, double x, double y,
                best = maxPrice - curMinPrice ;
                worst= minPrice - curMaxPrice ;
                if ( best >= 0 ){/* draw circle above */
-                  gl_print(&gl_smallFont, x + (sys->pos.x+11) * map_zoom , y + (sys->pos.y-22)*map_zoom, &cGreen, "%.1f",best);
-                  best = tanh ( best / 100. ) /2 ;
-                  /*ccol.r=best+0.5; ccol.g=best+0.5; ccol.b=0.5-best; ccol.a=1;*//*yellow*/
-                  ccol.r=1-2*best; ccol.g=best+0.5; ccol.b=0; ccol.a=1;/*green to orange*/
+                  gl_print(&gl_smallFont, x + (sys->pos.x+11) * map_zoom , y + (sys->pos.y-22)*map_zoom, &cBlue, "%.1f",best);
+                  best = tanh ( 10*best / curMinPrice );
+                  ccol.r=1-best;ccol.g=1-best;ccol.b=best;ccol.a=1;/*yellow (0) to blue (1)*/
```

Comment:
This is practically unreadable. You really should break these up into separate lines and give more space.

```
ccol.r = 1 - best;
ccol.g = 1 - best;
ccol.b = best;
ccol.a = 1;
```

This is a general criticism by the way, I've seen a lot of code from you that compacts down code far too much at the expense of readability, and I've corrected dozens of these. A few bytes are not worth making the code harder to read, especially because (in the words of the Python style guide):

> code is read much more often than it is written.

Reply:
Changed to a macro.